ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -757,6 +758,25 @@ periodics:
 
         # debug output
         oc describe -n ${namespace} istag/machine-os-content:latest
+
+        # Craft a release image
+        MOC_SHA256=$( oc get istag -n "${namespace}" istag/machine-os-content:latest -o template='{{ .image.metadata.name }}' )
+        export RELEASE_IMAGE_LATEST=registry.svc.ci.openshift.org/${namespace}/release:latest
+        export IMAGE_TESTS=registry.svc.ci.openshift.org/origin/$(BRANCH):tests
+        oc registry login
+        oc adm release new --from-release=registry.svc.ci.openshift.org/origin/release:$(BRANCH) \
+          --to-image=${RELEASE_IMAGE_LATEST} \
+          machine-os-content=registry.svc.ci.openshift.org/${namespace}/machine-os-content@${MOC_SHA256}
+
+        # execute the test
+        ci-operator \
+          --artifact-dir=$(ARTIFACTS) \
+          --namespace=${namespace} \
+          --secret-dir=/usr/local/pull-secret \
+          --secret-dir=/usr/local/e2e-$(CLUSTER_TYPE)-cluster-profile \
+          --target=e2e-$(CLUSTER_TYPE)
+
+        # promote to fcos namespace
         oc tag ${namespace}/machine-os-content:latest fcos/machine-os-content:${BRANCH}
         oc tag fcos/machine-os-content:${BRANCH} ${TAG_NS}/${BRANCH}:machine-os-content
```

Comment:
Change this line to come from the same source as the previous line.  This is a bug

Reply:
Updated to `${namespace}/machine-os-content@${MOC_SHA256}`